src/dispatch/case/service.py

```diff
@@ -69,7 +69,7 @@ def get_by_name_or_raise(*, db_session, project_id: int, case_in: CaseRead) -> C
 
 def get_all(*, db_session, project_id: int) -> List[Optional[Case]]:
```

Comment:
`List` and `Optional` are both deprecated. `List` -> `list` (no import required) and `Optional` becomes `Case | None` (no import required)

recommend we use this syntax going forward for consistency.

Reply:
Acknowledged. I've created a separate PR to handle the List/Optional/Union changes.
https://github.com/Netflix/dispatch/pull/4744